### PR TITLE
Moving loading data from forward models to python

### PR DESCRIPTION
**Pull request by ManInFez**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by ManInFez** [comment]
Current state of fixing unit-tests:
-- wrong data in test-data -- tests/unit_tests/test_libres_facade.py
       -- also libres.load_all_misfit_data() must be fixed

-- more investigation needed -- tests/unit_tests/data/test_integration_data.py
-- unsure -- tests/unit_tests/lsf_queue/test_lsf_driver.py

-- remove test -- tests/unit_tests/c_wrappers/res/enkf/test_summary_key_set.py
-- remove test -- tests/unit_tests/c_wrappers/res/enkf/data/test_gen_data.py

-- inconsistent timemap -- tests/unit_tests/cli/test_integration_cli.py
-- inconsistent timemap -- tests/unit_tests/c_wrappers/res/enkf/test_load_forward_model.py
-- inconsistent timemap --tests/unit_tests/status/test_tracking_integration.py

FIXED: -- using node.load tests/unit_tests/simulator/test_batch_sim.py
FIXED: -- rerun mini ert example with new storage -- tests/unit_tests/c_wrappers/res/enkf/test_enkf_main.py
FIXED: -- fetch gen data directly from enkf_fs -- tests/unit_tests/c_wrappers/res/enkf/test_simulation_batch.py
FIXED: -- fix data in test, write summary directly -- tests/unit_tests/c_wrappers/res/enkf/test_row_scaling_case.py
FIXED (ish): -- easy fix , write totalchi2 in python with pandas dataframes -- tests/unit_tests/all/plugins/test_export_misfit.py

File: src/clib/lib/enkf/enkf_state.cpp [hunk 1]
```diff
@@ -35,53 +35,22 @@ void enkf_state_initialize(enkf_fs_type *fs, enkf_node_type *param_node,
...
-    ecl_sum_type *summary = NULL;
-
+std::string get_ecl_header_file(std::string run_path, std::string eclbase) {
     char *header_file = ecl_util_alloc_exfilename(
-        run_path, eclbase, ECL_SUMMARY_HEADER_FILE, DEFAULT_FORMATTED, -1);
+        run_path.c_str(), eclbase.c_str(), ECL_SUMMARY_HEADER_FILE,
+        DEFAULT_FORMATTED, -1);
+    if (header_file == NULL)
+        return "";
+    return std::string(header_file);
```
Comment (pinkwah) [inline src/clib/lib/enkf/enkf_state.cpp:44]: `header_file` not free'd.

File: src/clib/lib/enkf/enkf_state.cpp [hunk 2]
```diff
@@ -35,53 +35,22 @@ void enkf_state_initialize(enkf_fs_type *fs, enkf_node_type *param_node,
...
 
-ecl_sum_type *load_ecl_sum(const char *run_path, const char *eclbase) {
-    ecl_sum_type *summary = NULL;
-
+std::string get_ecl_header_file(std::string run_path, std::string eclbase) {
     char *header_file = ecl_util_alloc_exfilename(
-        run_path, eclbase, ECL_SUMMARY_HEADER_FILE, DEFAULT_FORMATTED, -1);
+        run_path.c_str(), eclbase.c_str(), ECL_SUMMARY_HEADER_FILE,
+        DEFAULT_FORMATTED, -1);
+    if (header_file == NULL)
```
Comment (pinkwah) [inline src/clib/lib/enkf/enkf_state.cpp:42]: Should probably be an exception

File: src/ert/_c_wrappers/enkf/enkf_fs.py [hunk 3]
```diff
@@ -178,6 +181,52 @@ def copy_from_case(
             active,
         )
 
+    def save_summary_data(self, summary: "EclSum", realization: int):
+        import os
```
Comment (pinkwah) [inline src/ert/_c_wrappers/enkf/enkf_fs.py]: For this PR (pushed to its own feature-branch) use pandas dataframe + HDF5. Performance testing afterwards.

File: src/ert/_c_wrappers/enkf/enkf_state.py [hunk 4]
```diff
@@ -34,16 +37,22 @@ def _internalize_results(
...
-            run_arg.sim_fs,
-            run_arg.iens,
-        )
 
+        summary = EclSum(
+            f"{run_arg.runpath}/{run_arg.job_name}",
+            include_restart=False,
+            lazy_load=False,
+        )
+        run_arg.sim_fs.save_summary_data(summary, run_arg.iens)
```
Comment (pinkwah) [inline src/ert/_c_wrappers/enkf/enkf_state.py]: Get the numpy array out; don't pass EclSum as argument

File: src/ert/analysis/_es_update.py [hunk 5]
```diff
@@ -33,6 +39,17 @@ class ErtAnalysisError(Exception):
     pass
 
 
+@dataclass
+class UpdateSnapshot:
```
Comment (pinkwah) [inline src/ert/analysis/_es_update.py:42]: Pandas dataframe pls

**Pull request closed without merging**